hwdb/60-keyboard.hwdb

```diff
@@ -1475,6 +1475,14 @@ evdev:name:Sony Vaio Keys:dmi:bvn*:bvr*:bd*:svnSony*:pnVPC*:pvr*
  KEYBOARD_KEY_0d=down                                   # Fn+F9 zoomout
  KEYBOARD_KEY_0e=up                                     # Fn+F10 zoomin
 
+###########################################################
+# System76
+###########################################################
+
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnSystem76*:pn*
```

Comment:
This pattern is very generic. I don't think we can say **all** System76 laptops will always have the same keyboard. Please add a match also for pn. (See for example the patterns for Sony above this).

Reply:
I work at System76. Identifying by product name would involve matching a couple dozen products - the majority of our product line uses these codes and the rest simply do not output them so there is no risk of misinterpretation

Reply:
Here is what we install on all System76 products right now: https://github.com/pop-os/system76-dkms/blob/master/lib/udev/hwdb.d/99-system76-dkms.hwdb

We simply want to upstream it.